centaur/src/main/resources/standardTestCases/drs_usa_hca.test

```diff
@@ -16,15 +16,15 @@ metadata {
   status: Succeeded
 
   "outputs.drs_usa_hca.path1" =
-    "/cromwell_root/drs.data.humancellatlas.org/4cf48dbf-cf09-452e-bb5b-fd016af0c747/eb32bfc6-e7be-4093-8959-b8bf27f2404f.zarr!.zattrs"
+    "/cromwell_root/drs.data.humancellatlas.org/4cf48dbf-cf09-452e-bb5b-fd016af0c747_version_2019-09-14T024754.281908Z/eb32bfc6-e7be-4093-8959-b8bf27f2404f.zarr!.zattrs"
   "outputs.drs_usa_hca.path2" =
-    "/cromwell_root/drs.data.humancellatlas.org/fe6729a0-9d24-4034-a0e8-720830044af2/empty_drops_result.csv"
+    "/cromwell_root/drs.data.humancellatlas.org/fe6729a0-9d24-4034-a0e8-720830044af2_version_2019-09-14T075241.272244Z/empty_drops_result.csv"
   "outputs.drs_usa_hca.path3" =
-    "/cromwell_root/drs.data.humancellatlas.org/82544678-fb65-4da4-aaf8-3a2506b64993/empty_drops_result.csv"
+    "/cromwell_root/drs.data.humancellatlas.org/82544678-fb65-4da4-aaf8-3a2506b64993_version_2019-09-14T080840.009338Z/empty_drops_result.csv"
   "outputs.drs_usa_hca.path4" =
-    "/cromwell_root/drs.data.humancellatlas.org/d205fbd5-0b2d-4ec9-a898-ee08712e9c18/empty_drops_result.csv"
+    "/cromwell_root/drs.data.humancellatlas.org/d205fbd5-0b2d-4ec9-a898-ee08712e9c18_version_2019-09-14T024945.734484Z/empty_drops_result.csv"
   "outputs.drs_usa_hca.path5" =
-    "/cromwell_root/drs.data.humancellatlas.org/39ae3998-72a4-41ba-b00f-526bfecb8ca2/Retina_Wong_scRNA_Sample1_I1.fastq.gz"
+    "/cromwell_root/drs.data.humancellatlas.org/39ae3998-72a4-41ba-b00f-526bfecb8ca2_version_2019-09-13T091626.837580Z/Retina_Wong_scRNA_Sample1_I1.fastq.gz"
```

Comment:
Do we have any more adventurous real-world test cases than "has an underscore"?

Reply:
Not that I'm aware of